{{cookiecutter.project_name}}/server/main_app/templates/main_app/index.html

```diff
@@ -9,6 +9,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="{% static 'main_app/css/index.css' %}">
+
+    <link rel=“shortcut icon” type=“image/png” src=“{% static
```

Comment:
There are several things wrong with this statement:
1. you are using wrong quotes here: `src=“{%`. It should be `src="{%`
2. you are using wrong quotes here: `“/media/logo.jpg”`. It should be `'/images/favicon.png'`
3. you are using wrong directory name: `media`. It should be: `images`
4. you are using wrong filename: `logo`. It should be: `favicon-RESOLUTIONxRESOLUTION`
5. you are using wrong extension: `jpg`. It should be: `png`